tests/utils/utils.go

```diff
@@ -944,3 +947,82 @@ func SendExampleCrossChainMessageAndVerify(
 		Expect(currMessage).ShouldNot(Equal(message))
 	}
 }
+
+func InitChainConfig(
```

Comment:
Could you add a comment documenting what these functions do and/or what they are intended to be used for?

Reply:
Added